README.md

```diff
@@ -14,6 +14,43 @@ Install using
 
 I added code to wait for a flag to be true, which gives you finer control over when you are ready to publish your content.
 
+### Optionally set `Spiderable.userAgentRegExps`
+`Spiderable.userAgentRegExps` is array of strings, of bot user agents that we want to serve statically, but do not obey the _escaped_fragment_ protocol.
+```javascript
+Spiderable.userAgentRegExps.push(/^bingbot/i);
+```
+
+Default bots:
+ - `/^facebookexternalhit/i`
+ - `/^linkedinbot/i`
+ - `/^twitterbot/i`
+
+### Optionally set `Spiderable.skipRoutes`
+`Spiderable.skipRoutes` is array of strings, routes that we want to serve statically, but do not obey the _escaped_fragment_ protocol. For more info see this [thread](https://github.com/meteor/meteor/issues/3853).
+```javascript
+Spiderable.skipRoutes.push('/cdn/storage/Files/');
```

Comment:
@jazeee just a sample code, we can change to  something like `/path/to/server/route`, what do you will say?